apps/src/code-studio/pd/workshop_dashboard/attendance/workshop_attendance.jsx

```diff
@@ -184,25 +167,11 @@ const WorkshopAttendance = React.createClass({
   },
 
   render() {
-    if (this.state.loading) {
+    if (this.state.loadingSummary) {
       return <i className="fa fa-spinner fa-pulse fa-3x" />;
```

Comment:
Yeah I've thought about that. It's only one line so it's been a low priority, but I agree it should be its own component.